apps/microtvm/zephyr/demo_runtime/boards/qemu_riscv32.conf

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This file is specific to the QEMU-emulated RISCV32 microTVM board.
+
+# For TVMPlatformGenerateRandom(). Remember, these values do not need to be truly random.
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_TIMER_RANDOM_GENERATOR=y
+
+# Default 512, for operations with large floating point data. 
```

Comment:
why 512?

Reply:
based on my understanding from their doc, the default it 512. [Here](https://docs.zephyrproject.org/2.5.0/reference/kconfig/CONFIG_MAIN_STACK_SIZE.html?highlight=config_main_stack_size#cmdoption-arg-CONFIG_MAIN_STACK_SIZE) it mentioned for RISCV is 512.

Reply:
ok sounds good

Reply:
Done!